test/integration/next-image-new/invalid-image-import/test/index.test.ts

```diff
@@ -17,19 +17,33 @@ const appDir = join(__dirname, '../')
 let appPort: number
 let app
 let stderr = ''
-const msg =
-  'Error: Image import "../public/invalid.svg" is not a valid image file. The image may be corrupted or an unsupported format.'
 
 function runTests({ isDev }) {
   it('should show error', async () => {
     if (isDev) {
       const browser = await webdriver(appPort, '/')
       expect(await hasRedbox(browser)).toBe(true)
       expect(await getRedboxHeader(browser)).toBe('Failed to compile')
-      expect(await getRedboxSource(browser)).toBe(`./pages/index.js:3\n${msg}`)
-      expect(stripAnsi(stderr)).toContain(msg)
+      const source = await getRedboxSource(browser)
+      if (process.env.TURBOPACK) {
+        expect(source).toMatchInlineSnapshot(`
+          "./test/integration/next-image-new/invalid-image-import/public/invalid.svg
+          Processing image failed
+          Failed to parse svg source code for image dimensions
+
+          Caused by:
+          - Source code does not contain a <svg> root element"
```

Comment:
So much better ;)

Reply:
Yeah! Can still be better but it's a small improvement 🙌